doc/source/user_guide/solver_settings/solver_settings_contents.rst

```diff
@@ -132,7 +132,7 @@ and ``NamedObject`` types, the state value is a dictionary. For the
   >>> viscous = pyfluent.solver.Viscous(settings_source=solver)
   >>> viscous.model = 'laminar'
   >>> energy = pyfluent.solver.Energy(settings_source=solver)
-  >>> energy = { 'enabled' : False }
```

Comment:
@hpohekar, could you please confirm if this is working.

Reply:
Done. already.

Reply:
@hpohekar, Is the dictionary style assignment working?

Reply:
It's removed because it is not working for built in settings objects. It works as follows:

`>>> solver.settings.setup.models.energy = {"enabled": True}`

Reply:
Understood, Thank you for clarifying.

Reply:
@prmukherj It is not expected to work as `energy = { 'enabled' : False }` because it is reassigning energy object to dict object.

Reply:
Maybe we need to reconsider https://github.com/ansys/pyfluent/issues/85. Obviously, this is a trap for pyFluent developers to fall into, so what chance to do Fluent users without Python knowledge have??

If we're going to deprecate that assignment style for API objects then we should do it before progressing from 0.x to 1.x.

Reopening the issue.

Reply:
@seanpearsonuk Yes, I agree.